lya/data.py

```diff
@@ -38,6 +43,17 @@ def plot_pk(params=None, filename=lya_data_path):
     fig : matplotlib Figure instance
     """
 
+    data = load_lya_data(filename)
+
+    if params is None:
```

Comment:
won't the following few lines have to be repeated if `params` is passed?  might be better to test for `if params is not None:` after plotting the data (which it will always do).  Then @aryanrahimieh, you fill in what needs to happen if `params` is passed.

Reply:
yeah, I was just being lazy :D 

I put it after because I realized the numbers passed to subplots will be different if there is a new panel, so I thought I would leave it to Aryan to figure it out :D 

Is this ok or do you want me change it?

Reply:
We can leave it as is, and if Aryan finds himself duplicating code then he can switch things around.